pyanaconda/modules/payload/live/live_os_interface.py

```diff
@@ -64,3 +70,35 @@ def SetupInstallationSourceWithTask(self) -> ObjPath:
     def TeardownInstallationSourceWithTask(self) -> ObjPath:
         """Teardown installation source resources."""
         return self.implementation.teardown_installation_source_with_task()
+
+    def PreInstallWithTask(self) -> ObjPath:
```

Comment:
Couldn't we move these methods to a base interface?

Reply:
Yes, I want to do that in a future PR.